packages/modals/src/styled/StyledModal.ts

```diff
@@ -47,14 +47,26 @@ const animationStyles = (props: IStyledModalProps) => {
   return '';
 };
 
-const boxShadow = (props: ThemeProps<DefaultTheme>) => {
-  const { theme } = props;
-  const { space, shadows } = theme;
-  const offsetY = `${space.base * 5}px`;
-  const blurRadius = `${space.base * 7}px`;
-  const color = getColorV8('neutralHue', 800, theme, 0.35);
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const offsetY = `${theme.space.base * 5}px`;
+  const blurRadius = `${theme.space.base * 7}px`;
```

Comment:
The spread is different for dark mode. Please compare with in-progress branch: https://github.com/zendeskgarden/react-components/blob/9f82a0296642f4f10ab6b5327f904d71f2aa519d/packages/theming/src/utils/menuStyles.ts#L67-L71

Reply:
Good catch! 💯 There is difference in Dark mode. I'll update. 

Unrelated... but should we raise these 1px shadow spread differences with design? They seem questionable. 🤔

Reply:
Already did 😉; @lucijanblagonic confirmed the difference is intentional.